README.md

```diff
@@ -3,10 +3,19 @@ A curated list of precompiled Wasm binaries of
 programs that are known to successfully work with
 [Wasm sandbox](https://github.com/golemfactory/sp-wasm) in
 [Golem](https://github.com/golemfactory/golem).
+The list includes applications directly located in this repo,
+as well as links that points to external.
+The applications can be raw or augmented with a client.
+When using raw, a user connects directly with Golem. 
+That is more technical approach.
 See [here](https://docs.golem.network/#/About/Use-Cases?id=wasm)
 to learn how to launch a Wasm task in Golem.
+The applications augmented with a client are more user friendly,
+because it handles communication with Golem node,
+but there is no generic client. See g-flite for instance.
 
 The list:
+* [g-flite](https://github.com/golemfactory/g-flite) - text-to-speech
```

Comment:
@lukasz-glen How about we create 2 lists then: 1) with raw WASM binaries, and 2) with GUI/CLI client apps such as `g-flite`?

Reply:
sounds good